docker-compose.yml

```diff
@@ -1,8 +1,47 @@
 version: "2"
 services:
-  app:
-    image: "swisstopo/${SERVICE_NAME}:local"
+  s3:
+    image: minio/minio
+    env_file: ./minio.env
+    command: server /data
+    volumes:
+      - type: bind
+        source: ${PWD}/.volumes/minio
+        target: /data
```

Comment:
Update the version to at least 3.4. In version 2 the volumes don't support the object syntax like here but only the short syntax (`- ${PWD}/.volumes/minio:/data`)